app/controllers/users/registrations_controller.rb

```diff
@@ -9,8 +9,11 @@ def send_devise_notification(notification, *args)
     def edit; end
 
     def update
-      if user_params[:password].blank? ? @user.update_without_password(user_params) : @user.update_with_password(user_params)
-        redirect_to edit_user_registration_path, notice: I18n.t('activerecord.attributes.user.successful_update')
+      if user_params[:password].blank? ?
+        @user.update_without_password(user_params) :
+        @user.update_with_password(user_params)
```

Comment:
it's not okay, firstly don't use if in condition and then don't use ternary operator for two or more lines